bowtie/templates/report.html.j2

```diff
@@ -410,13 +419,93 @@
+            data-bs-custom-class="custom-tooltip"
+            data-bs-title="Copy to clipboard">
+            {# Clipboard SVG Icon  #}
+              <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-clipboard" viewBox="0 0 16 16">
+                <path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z"/>
+                <path d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z"/>
+              </svg>
+            </button>
+            <button type="button" class="btn text-nowrap" onclick="closeSchemaCard()" aria-label="Close" data-bs-toggle="tooltip" data-bs-placement="top"
+            data-bs-custom-class="custom-tooltip"
+            data-bs-title="Close">
+            {# Close SVG Icon  #}
+              <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-x-square" viewBox="0 0 16 16">
+                <path d="M14 1a1 1 0 0 1 1 1v12a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1h12zM2 0a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2H2z"/>
+                <path d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z"/>
+              </svg>
+            </button>
+          </div>
+      </div>
+      <div id="code" class="card-body"></div>
+    </div>
+  {# Schema Display end  #}
   {% endfor %}
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
   <script>
     const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="tooltip"]')
     const tooltipList = [...tooltipTriggerList].map(tooltipTriggerEl => new bootstrap.Tooltip(tooltipTriggerEl))
   </script>
-
+  {# Scripts for Schema Display start  #}
+  <script>
+      const displayBody = document.querySelector("#display-body");
+      const codes = document.querySelector("#code");
+      function displayCode(code) {
+        displayBody.classList.remove("d-none");
+        displayBody.classList.add("d-block");
+        codes.innerHTML = `<pre>${code}</pre>`;
+      }
+      function schemaDisplay(parentBodyId, code) {
+        displayBody.classList.remove("d-none");
+        displayBody.classList.add("d-block");
+        displayCode(code);
+        const isDisplayed = displayBody.parentElement === document.getElementById(parentBodyId);
+        // If the code segment is displayed, remove it
```

Comment:
A lot of the comments above are fine/definitely improvements for understanding SVG icons, but these kind of comments seem less so, they just repeat what some fairly simple code seems to be doing, or repeat the name of a function.

Reply:
Removed some of the comments